backend/cellguide/pipeline/ontology_tree/__init__.py

```diff
@@ -5,11 +5,12 @@
 )
 from backend.cellguide.pipeline.ontology_tree.tree_builder import OntologyTreeBuilder
 from backend.cellguide.pipeline.utils import output_json
+from backend.wmg.api.wmg_api_config import WMG_API_SNAPSHOT_SCHEMA_VERSION
 from backend.wmg.data.snapshot import WmgSnapshot, load_snapshot
 
 
-def run(output_directory):
-    snapshot: WmgSnapshot = load_snapshot()
+def run(output_directory) -> OntologyTreeBuilder:
+    snapshot: WmgSnapshot = load_snapshot(snapshot_schema_version=WMG_API_SNAPSHOT_SCHEMA_VERSION)
```

Comment:
i ran mypy against the cellguide pipeline module and this was one of the errors it found. we need to specify the schema version in `load_snapshot`, though you could argue that maybe `load_snapshot` should just default to the constant that's specified in `wmg_api_config`